cmd/access/main.go

```diff
@@ -109,31 +109,45 @@ func main() {
 		flags.StringVar(&anb.stakedAccessNodeIDHex, "staked-access-node-id", "", "the node ID of the upstream staked access node if this is an unstaked access node")
 		flags.StringVar(&anb.unstakedNetworkBindAddr, "unstaked-bind-addr", cmd.NotSet, "address to bind on for the unstaked network")
 	})
-	anb.Initialize().
-		Module("mutable follower state", func(node *cmd.FlowNodeBuilder) error {
+
+	// initialize the node builder to parse the command line args
+	anb.Initialize()
+
+	// choose a staked or an unstaked node builder based on anb.staked
```

Comment:
At this point, we know what type of access node this is (staked or unstaked) and we then proceed with the appropriate node builder.